src/services/harry-potter.js

```diff
@@ -0,0 +1,27 @@
+export async function getCharacters() {
+  const response = await fetch('https://hp-api.herokuapp.com/api/characters/staff');
+
+  const data = await response.json();
+
+  let uniqueId = 0;
+  data.forEach((char) => {
```

Comment:
Mapping through this and using the second argument to `.map` the index might be an easier way to do this
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map
Both of these solutions however will end up with the item's id being equivalent to it's index value which should be avoided in React
https://reactjs.org/docs/lists-and-keys.html